p2p/transport/webrtc/transport.go

```diff
@@ -76,6 +77,7 @@ const (
 // timeout values for the peerconnection
 // https://github.com/pion/webrtc/blob/v3.1.50/settingengine.go#L102-L109
 const (
+	receiveMTUBytes            = 1500
```

Comment:
Pion defaults to 1460 which sounds correct:

https://github.com/pion/webrtc/blob/c259e89eb53b6d599c9dc7ec3e259fdb26e15f95/constants.go#L9-L10

Can you add a comment please (or remove it entirely if it's not needed) ?

Reply:
This failed once with ErrShortBuffer when the MTU was only 1460. But I think that's because this has a higher MTU and we should remove both the settings(https://github.com/libp2p/go-libp2p/blob/master/p2p/transport/webrtc/udpmux/mux.go#L20)

Reply:
I've removed this for now. I'll investigate this when this fails.

Reply:
Dumb question but shouldn't udpmux also use 1460 MTU ?

Reply:
I think so. Will take that in a separate PR.

Reply:
This doesn't matter. It's just used for receiving and not for deciding the packet size on the write path. Renamed the variable and added a comment.

Reply:
I think I found the bug:
This was causing pion to send larger packets some times. 
https://github.com/pion/sctp/pull/301/files